src/vs/workbench/contrib/terminal/common/scripts/shellIntegration.ps1

```diff
@@ -169,6 +169,18 @@ elseif ((Test-Path variable:global:GitPromptSettings) -and $Global:GitPromptSett
 	[Console]::Write("$([char]0x1b)]633;P;PromptType=posh-git`a")
 }
 
+
+if (-not (Get-Module -Name PSReadLine)) {
+	# pwsh version 7+ is required for accessibility support in PSReadLine
```

Comment:
I just want to clarify where this requirement is coming from? I don't know of a technical reason beyond 5.1 being a lot less reliable than 7 and feature-frozen.